inception-workload-dynamic/NOTICE.txt

```diff
@@ -0,0 +1,4 @@
+The following icon sets are licensed under a Creative Commons Attribution 3.0 License.
+Source: http://www.fatcow.com/free-icons
+
+book_open.png
```

Comment:
I'd suggest we use font-awesome instead of icons. We use font-awesome in most places in the UI now.